test/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ if(SINGULARITY_USE_EOSPAC AND SINGULARITY_TEST_SESAME)
                         PRIVATE Catch2::Catch2 singularity-eos::singularity-eos)
 endif()
 
-if(SINGULARITY_BUILD_CLOSURE)
+if(SINGULARITY_BUILD_CLOSURE AND SINGULARITY_USE_SPINER)
```

Comment:
I think we should add fortran to the if list.

Reply:
The closure code is built independent of Fortran, `singularity-eos/CMakeLists.txt`

```cmake
if (SINGULARITY_BUILD_CLOSURE)
  list(APPEND EOS_SRCS eos/singularity_eos.cpp)
  list(APPEND EOS_HEADERS eos/singularity_eos.hpp closure/mixed_cell_models.hpp)
endif()

if (SINGULARITY_USE_FORTRAN)
  list(APPEND EOS_SRCS eos/singularity_eos.f90)
  # would rather handle this more robustly, being sloppy for now
  list(APPEND EOS_MODS singularity_eos.mod singularity_eos_types.mod)
endif()
```

Reply:
Yes, but you'd be saving me some time when I have to remember to do this in my PR b/c soon it'll only be built when also fortran.

Reply:
After looking at it closer, you're correct, this test can remain but other things will need to be ifdef'd out.

Reply:
Requested change rescinded.